scenario/scenario.py

```diff
@@ -222,14 +212,13 @@ def play(
         context: Context = None,
         add_to_playbook: bool = False,
     ) -> PlayResult:
+        scene = obj
```

Comment:
I had this in the conditional because it'd be hard to type this variable. This code is complex enough as it is, I don't think a bit extra explicitness can hurt.